### ISSUE TITLE
Images with black/white background

### ISSUE THREAD

**Issue by giteeeeee** [nitrosocke/dreambooth-training-guide#5]
Do you suggest to use subject images with black/white background BUT also one image with background scenes? Am I understanding correctly?

**Comment by nitrosocke** [comment]
Yes that's correct. I found that subjects on black or white background train very well and fast. After discovering that I tried a training with only images of objects on black and even after 5x the normal training steps I'd use the model still couldn't render the objects.
After that I did the digimon test model you can see in the article and it worked immediately.

**Comment by giteeeeee** [comment]
> Yes that's correct. I found that subjects on black or white background train very well and fast. After discovering that I tried a training with only images of objects on black and even after 5x the normal training steps I'd use the model still couldn't render the objects. After that I did the digimon test model you can see in the article and it worked immediately.

Thanks for the reply.
So how exactly did you change the input images in the digimon test? Giving half of the images a background scene while the other half still in plain black background?

**Comment by Humanoidme** [comment]
So is it black or white ? or can it be mixed with black and white backgrounds?

**Comment by myndxero** [comment]
Just wanted to point out why there's an outline in your photos with transparent backgrounds.  Any pixel thatt has a partial transparency will turn white. If you edit your own pictures, mind the tolerance of your selection tools. You don't want any partial transparent pictures.  You can check this by saving the picture to a format that doesn't support transparencies.  Hope this helps!
